### PR TITLE
[#51, #52] Use 'you' and Sentence case

### PR DESCRIPTION
**Pull request by Tejas2805**
Fixes part of #51 and #52 

* [x] About this Book
* [x] OOP
* [X] Software Engineering
* [X] Requirements
* [x] Implementation
* [x] Design

```
Let's follow Google's documentation guidelines, to make the 
book more consistent.
```

### CODE REVIEW

**Comment by Tejas2805** [comment]
Have done the following changes:

1. Replace 'we' with 'you'.
1. Modify sentence containing 'we' a bit to fit the 'you' context
1. Use sentence case for headings based on Google documentation.
1. Add EOF lines to files that don't have them.

**Comment by Tejas2805** [comment]
@damithc Any specific reason for the deploy to fail? Otherwise, the PR is ready for review.

**Comment by Tejas2805** [comment]
@damithc Can I get your review on this? Will do the second part accordingly.

File: oop/associations/what/text.md [hunk 1]
```diff
@@ -26,15 +26,15 @@ Objects in an OO solution need to be connected to each other to form a network s
 
 <box>
 
-{{ icon_example }} In our example, as some `Course` objects can have associations with some `Student` objects, we can view it as an association between the `Course` class and the `Student` class.
+{{ icon_example }} In our example, as some `Course` objects can have associations with some `Student` objects, youcan view it as an association between the `Course` class and the `Student` class.
```
Comment (damithc) [inline oop/associations/what/text.md:29]: missing space

**Comment by damithc** [comment]
Note that in general it is better to fix only one issue per PR, unless the issues are inter-dependent.

**Comment by Tejas2805** [comment]
> Note that in general it is better to fix only one issue per PR, unless the issues are inter-dependent.

Ya agreed. It's just that since I was going through all the files one by one, I went ahead with both. Will split up for the second part.

**Comment by Tejas2805** [comment]
@damithc Can merge this, if all okay?

File: oop/associations/what/text.md (hunk 1 above)
Comment (damithc) [reply oop/associations/what/text.md:29]: @Tejas2805 Have you fixed this?

Comment (Tejas2805) [reply oop/associations/what/text.md:29]: Oh fixed it that day itself. But due to shifting from hostel last 2/3 days forgot to inform earlier. 

Can see the change [here](https://github.com/se-edu/se-book/pull/83/commits/e92d454b31bd9a0222975b5e24e1adb3bdee224b)

Comment (damithc) [reply oop/associations/what/text.md:29]: Why is it not showing in the PR diff?

File: oop/associations/what/text.md [hunk 2]
```diff
@@ -26,15 +26,15 @@ Objects in an OO solution need to be connected to each other to form a network s
 
 <box>
 
-{{ icon_example }} In our example, as some `Course` objects can have associations with some `Student` objects, we can view it as an association between the `Course` class and the `Student` class.
+{{ icon_example }} In our example, as some `Course` objects can have associations with some `Student` objects, you can view it as an association between the `Course` class and the `Student` class.
```
Comment (Tejas2805) [inline oop/associations/what/text.md:29]: @damithc Looks okay now. Can check.

**Pull request merged**